src/scripts/db_script.sql

```diff
@@ -0,0 +1,19 @@
+-- DATABASE : ea_restaurant_db
+
+CREATE DATABASE ea_restaurant_db
+
+-- Table : products
+
+CREATE TABLE IF NOT EXISTS products
+									(
+    									product_id serial NOT NULL ,
+    									product_name varchar(100) NOT NULL,
+    									product_description varchar(50) default 'no description',
+    									entity_status varchar(50) NOT NULL,
+    									create_by integer NOT NULL,
+										create_date timestamp without time zone,
+    									update_by integer,
```

Comment:
This one should be not null as well since at insertion created by and updated by could be the same

Reply:
done